packages/async_graphics/async_graphics.0.6.0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "Leo White <leo@lpw25.net>"
+authors: ["Leo White <leo@lpw25.net>"]
+homepage: "https://github.com/lpw25/async_graphics"
+bug-reports: "https://github.com/lpw25/async_graphics/issues"
+dev-repo: "https://github.com/lpw25/async_graphics.git"
+license: "LGPL-2.0 with OCaml linking exception"
+tags: [
+  "async"
+  "graphics"
+]
+remove: [["ocamlfind" "remove" "async_graphics"]]
+depends: [
+  "ocamlfind"
```

Comment:
Could you add the `{build}` attribute to `ocamlfind` ? It seems that your package doesn't use the `Findlib` library.

Reply:
Done